api/src/search/backend/load_opportunities_to_index.py

```diff
@@ -163,12 +178,12 @@ def _handle_incremental_upserts(self, existing_opportunity_ids: set[int]) -> Non
             loaded_ids = self.load_records(opportunities_to_index)
             logger.info(f"Indexed {len(loaded_ids)} opportunities")
 
-        # Clear processed / skipped entries from the queue
-        self.db_session.execute(
-            delete(OpportunitySearchIndexQueue).where(
-                OpportunitySearchIndexQueue.opportunity_id.in_(processed_opportunity_ids)
+            # Update updated_at timestamp instead of deleting records
+            self.db_session.execute(
+                update(OpportunityChangeAudit)
+                .where(OpportunityChangeAudit.opportunity_id.in_(processed_opportunity_ids))
+                .values(updated_at=get_now_us_eastern_datetime())
```

Comment:
In the DB, we use UTC for everything, I would make this call the UTC function, not the eastern one